src/InformaticsGateway/Services/Common/OutputDataPluginEngine.cs

```diff
@@ -53,6 +55,8 @@ public async Task<ExportRequestDataMessage> ExecutePlugInsAsync(ExportRequestDat
                 throw new PlugInInitializationException("InputDataPlugInEngine not configured, please call Configure() first.");
             }
 
+            var test = _serviceProvider.GetService<IRemoteAppExecutionRepository>();
```

Comment:
We shall avoid referencing plug-ins directly.

Reply:
will remove, just for temp testing